public/components/content-list-drawer/_content-list-drawer.scss

```diff
@@ -72,6 +72,26 @@
         font-weight: bold;
         padding-bottom:10px;
         border-bottom:4px solid $c-grey-300;
+
+        &--closed {
+            writing-mode: vertical-lr;
```

Comment:
Yeah, this isn't great and the click area is messed up. I'm going to have a look at refactoring it now.